src/Model/Payment/Pix.php

```diff
@@ -46,16 +47,13 @@ class Pix extends AbstractPayment implements PaymentInterface
     /**
      * @return string
      */
-    public function getImage()
-    {
-        return esc_url(Core::plugins_url('assets/images/pix.svg'));
-    }
-
-    /**
-     * @return string|null
-     */
-    public function getMessage()
+    public static function getDefaultCheckoutInstructions()
     {
-        return __('O QR Code para seu pagamento através de PIX será gerado após a confirmação da compra. Aponte seu celular para a tela para capturar o código ou copie e cole o código em seu aplicativo de pagamentos.', 'woo-pagarme-payments');
+        return __(
+            'The QR Code for your payment through PIX will be generated after confirming the purchase. '
```

Comment:
The QR Code for your payment `with` PIX will be generated [...]

Reply:
Ajustado